docs/OV_Runtime_UG/performance_hints.md

```diff
@@ -0,0 +1,146 @@
+
+The hints also "reverse" the direction of the configuration in the right fashion: rather than map the application needs to the low-level performance settings, and keep an associated application logic to configure each possible device separately, the idea is to express a target scenario with a single config key and let the *device* to configure itself in response.
+As the hints are supported by every OpenVINO device, this is completely portable and future-proof solution. 
+
+Previously, certain level of automatic configuration was coming from the _default_ values of the parameters. For example, number of the CPU streams was deduced from the number of CPU cores, when the `ov::streams::AUTO` (`CPU_THROUGHPUT_AUTO` in the pre-OpenVINO 2.0 parlance) is set. However, the resulting number of streams didn't account for actual compute requirements of the model to be inferred.
+The hints, in contrast, respect the actual model, so the parameters for the optimal throughput are calculated for each model individually (based on it's compute versus memory bandwidth requirements and capabilities of the device).
+
+## Performance Hints: Latency and Throughput
+As discussed in the [Optimization Guide](../optimization_guide/dldt_optimization_guide.md) there are few different metrics associated with the inference speed.
+Throughput and latency are some of the most critical factors that influence the overall performance of an application.
+
+This is why, to ease the configuration of the device, the OpenVINO already offers two dedicated hints, namely `ov::hint::PerformanceMode::THROUGHPUT` and `ov::hint::PerformanceMode::LATENCY`.
+Every OpenVINO device supports these, which makes the things portable and future-proof.
+The also allows to do a performance configuration that is fully compatible with the [automatic device selection](./auto_device_selection.md).
+A special `ov::hint::PerformanceMode::UNDEFINED` acts same just as specifying no hint.
+
+Please also see the last section in the document on conducting the performance measurements with the `benchmark_app`.
+
+Notice that if there are other performance factors (other than inference time) like memory footprint and model load/compilation time are of concern, a typical model may take significantly more time to load with `ov::hint::PerformanceMode::THROUGHPUT` and then consume  much more memory, compared to the `ov::hint::PerformanceMode::LATENCY`.  
+
+## Performance Hints: How It Works?
+Internally, every device "translates" the value of the hint to the actual performance settings.
+For example the `ov::hint::PerformanceMode::THROUGHPUT` selects number of CPU or GPU streams.
+For the GPU, additionally the optimal batch size is selected and the [automatic batching](../OV_Runtime_UG/automatic_batching.md) is applied whenever possible (and also [when the device supports that](./supported_plugins/Device_Plugins.md#features-support-matrix)).
```

Comment:
Maybe it would be nice to mention if user used -hint -tput  with -b 1, the -b 1  will  be ignored.

Reply:
-b 1 is never ignored. it is just no-op if the model is already batch1. this logic in the benchamrk_app doesn't relate to the hints